_config.yml

```
@@ -1,22 +1,25 @@
-original_repo: https://github.com/hackcu/live
-title: Local Hack Day LIVE
-color: "#23415a"
-secondary_color: "#8695a0"
-devpost: https://mlh-local-hack-day-2018.devpost.com/
-
-collections:
-  sections:
-    output: true
-    permalink: /:path/
-
-sass:
-    sass_dir: assets/_sass
-    style: compressed
-
-timezone: America/Denver
-
-# Countdown Timer
-# Time format: "MONTH DAY YEAR HH:MM:SS TIMEZONE"
-timer_enable: true
-start_time: "December 1 2018 09:00:00 GMT-0700"
-end_time: "December 1 2018 21:00:00 GMT-0700"
+original_repo: https://github.com/hackcu/live
+title: Hack CU V Live
+color: "#22292c"
+secondary_color: "#1AA8FF"
+devpost: https://hackcuv.devpost.com/
+
+collections:
+  sections:
+    output: true
+    permalink: /:path/
+
+sass:
+    sass_dir: assets/_sass
+    style: compressed
+
+timezone: America/Denver
+
+# Countdown Timer
+# Time format: "MONTH DAY YEAR HH:MM:SS TIMEZONE"
+timer_enable: true
+start_time: "February 23 2019 08:00:00 GMT-0700"

```

Hacking starts at 12
```suggestion
start_time: "February 23 2019 12:00:00 GMT-0700"
```